docs/reference_guide.md

```diff
@@ -204,13 +204,13 @@ iscsid is sleeping.
 It can also be made executable to run stand-alone. Start by adding an interpreter line at the top (`#!`) with either the path to your installed bpftrace (/usr/local/bin is the default) or the path to `env` (usually just `/usr/bin/env`) followed by `bpftrace` (so it will find bpftrace in your `$PATH`):
 
 ```
-     1	#!/usr/local/bin/bpftrace
-     1	#!/usr/bin/env bpftrace
-     2
-     3	tracepoint:syscalls:sys_enter_nanosleep
-     4	{
-     5	        printf("%s is sleeping.\n", comm);
-     6	}
+1 #!/usr/local/bin/bpftrace
```

Comment:
What's with the double 1's?

Reply:
I think it's to indicate that there are (at least) 2 shebang options and that you should only use one. But it is a bit confusing

Reply:
Yeah, it's too confusing. I'd ditch the env version.

Reply:
Fixed